src/components/CommentCard.jsx

```diff
@@ -0,0 +1,5 @@
+const CommentCard = () => {
```

Comment:
We don't really want to create all the components before we actually need them - commenting this once for all of them.